website/docs/integrations/home_assistant.md

```diff
@@ -66,10 +66,10 @@ Don't forget to replace `<teslamate url>` and `<your tesla model>` with correct
         payload_not_available: "false"
     device: &teslamate_device_info
       identifiers: [teslamate_car_1]
-      configuration_url: https://teslamate.zxxz.io/
+      configuration_url: https://teslamate.zxxz.io/ # update this with your teslamate URL
```

Comment:
@JakobLichterfeld - alternative suggestion

<teslamate url> # update this with your teslamate URL, e.g. https://teslamate.zxxz.io/

that way users can see search for the placeholders (same for the other placeholders).

but both changes fine for me

Reply:
You are absolutely right and I have been lazy. 🙂